packages/backend/src/config/makeConfig.ts

```diff
@@ -184,3 +192,68 @@ function getDiscordConfig(env: Env, isLocal?: boolean): DiscordConfig | false {
     }
   )
 }
+function getTokensInNewFormat(
```

Comment:
This is quite a long function, maybe it would make sense for it to go to it's own file